src/bitmessageqt/settings.py

```diff
@@ -1,16 +1,24 @@
 # -*- coding: utf-8 -*-
```

Comment:
Why do you need a second pass for this module? Something similar was already merged.

Reply:
Indeed these changes are already in.